packages/react-log-viewer/src/LogViewer/utils/constants.ts

```diff
@@ -0,0 +1,20 @@
+// Logger Dimensions
+export const LOGGER_HEIGHT: number = 600;
```

Comment:
Height, columns amount, all IDs, column widths, row height (and/or the row height function), min search input length might all be better off as props instead of constants. Anything a user may want to change on demand.

Width already is a prop so I'm not sure this constant gets used, same for row height which right now is calculated.